services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
```

Comment:
According to the checklist, you should use `get_user_model()` instead of directly using the `User` model. This is a best practice in Django to avoid potential issues with custom user models.